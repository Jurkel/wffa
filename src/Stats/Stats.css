.stats-wrapper {
  display: flex;
  flex-direction: column;
  background: transparent;
}

.stats-text {
  text-align: center;
  background: transparent;
}

.stats-wrapper h4 {
  color: #FFD08A;
  font-size: 14px;
  font-family: 'Courier';
  margin-bottom: -15px;
}

.stats-wrapper h3 {
  color: #f2f2f2;
  font-size: 15px;
  margin-bottom: 5px;
}

.stats-flex-row {
  display: inline-flex;
  flex-direction: row;
}

.stats-flex-row h4 {
  margin-right: 5px;
}

.wins-losses {
  display: flex;
  flex-direction: row;
  margin-left: 33%;
}

.points-scored h3, h4 {
  background: transparent;
}

.points-against h3, h4 {
  background: transparent;
}

.stats-wins {
  margin-right: 15px;
}

@media screen and (min-width: 1023px) {
  .wins-losses {
    margin-left: 36%;
  }
}