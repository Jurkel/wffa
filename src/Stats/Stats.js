import React from 'react';
import './Stats.css';

function Stats(props) {
  return (
    <div className='stats-wrapper'>
      <h2 className='player-title'>Stats</h2>
      <div className='stats-text'>
        <div className='wins-losses'>
          <div className='stats-wins'>
            <h4>Wins</h4>
            <h3>{props.rosterData.settings.wins}</h3>
          </div>
          <div className='stats-losses'>
            <h4>Losses</h4>
            <h3>{props.rosterData.settings.losses}</h3>
          </div>
        </div>
        <div className='waiver-budget stats-flex-row'>
          <h4>Waiver Budget Used: </h4><h3>${props.rosterData.settings.waiver_budget_used}</h3>
        </div>
        <br />
        <div className='total-aquisition stats-flex-row'>
          <h4>Waiver Position: </h4><h3>{props.rosterData.settings.waiver_position}</h3>
        </div>
        <div className='points-scored'>
          <h4>Points Scored</h4>
          <h3>{props.rosterData.settings.fpts}.{props.rosterData.settings.fpts_decimal}</h3>
        </div>
        <div className='points-against'>
          <h4>Points Against</h4>
          <h3>{props.rosterData.settings.fpts_against}.{props.rosterData.settings.fpts_against_decimal}</h3>
        </div>
      </div>
    </div>
  );
};

export default Stats;