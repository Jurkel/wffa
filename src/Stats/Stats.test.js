import React from "react";
import { shallow } from "enzyme";
import Stats from "./Stats";

it("renders Stats component", () => {
  const mockProps = {
    rosterData: {
      settings: {
        wins: 1,
        waiver_position: 10,
        waiver_budget_used: 45,
        total_moves: 0,
        ties: 0,
        ppts_decimal: 58,
        ppts: 231,
        losses: 1,
        fpts_decimal: 78,
        fpts_against_decimal: 78,
        fpts_against: 230,
        fpts: 214,
      },
    },
  };
  expect(shallow(<Stats {...mockProps} />)).toMatchSnapshot();
});
