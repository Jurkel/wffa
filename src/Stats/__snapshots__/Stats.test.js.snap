// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Stats component 1`] = `ShallowWrapper {}`;
